WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListFragment.java

```diff
@@ -213,14 +214,26 @@ public void onClick(View v) {
                                         }
                                     }).show();
                         } else if (savedLocally) {
-                            Snackbar.make(getActivity().findViewById(R.id.coordinator),
-                                    R.string.editor_post_saved_locally_not_published, Snackbar.LENGTH_LONG)
-                                    .setAction(R.string.button_publish, new View.OnClickListener() {
-                                        @Override
-                                        public void onClick(View v) {
-                                            publishPost(post);
-                                        }
-                                    }).show();
+                            if (isScheduledPost) {
```

Comment:
I have thought the same thing, but decided it's better to be consistent with the rest of the code, since it's all duplicating that block. For example, see the `else if (PostStatus.fromPost(post) == PostStatus.DRAFT) {` statement.

Looking at it again, I guess I can refactor the whole thing to be better.